### DISCUSSION TITLE
Dropdown always close when open

### DISCUSSION THREAD

**Discussion by Keroro2139** [heroui-inc/heroui#672] [Help] [answered]
Hi, I try to use Dropdown component, it seem like always close when i just click `Dropdown.Button`. But when I click then hold my mouse it seem like to work find :( .

https://user-images.githubusercontent.com/21120762/185272218-86842806-33dd-4ac5-974e-44ea7fd0dd0a.mov

```
<Dropdown disableAnimation placement="bottom-left">
    <Dropdown.Button light color="primary" ripple={false}>
        {t('common:sort')}
    </Dropdown.Button>
    <Dropdown.Menu
        css={{ w: 'auto' }}
        selectionMode="single"
        selectedKeys={selectedKeys}
        onSelectionChange={onSelectSort}
        items={items}
    >
        {(item: any) => (
            <Dropdown.Item
                key={item.key}
                css={{ fontSize: '.875rem' }}
                textValue={item.key}
            >
                {item.label}
            </Dropdown.Item>
        )}
    </Dropdown.Menu>
</Dropdown>
```

**Comment by Keroro2139** [comment]
I had changed from `yarn` to `npm`. I don't know why but it's work now. 
I'm using `react: 18.2.0` and `@nextui-org/react: 1.0.0-beta.9`.

1. Remove `node_modules` `.next` `yarn.lock`
2. Run `npm i` or `npm install`
3. Run `npm run dev`

But the animations still didn't work, so i just disable it first.

https://user-images.githubusercontent.com/21120762/185276510-24a25aee-b395-4147-a204-dfd1caf77913.mov

**Comment by cuevacelis** [comment]
Tengo el mismo problema, en localhost funciona bien pero en producción(vercel) no 👎

**Comment by jrgarciadev** [comment, marked as answer]
Hey guys, could you try again using the latest version? `v1.0.0-beta.10`

**Reply by cuevacelis** [reply]
Funciona bien en la ultima versión.

Gracias :+1:

**Reply by jrgarciadev** [reply]
Genial! 🚀
